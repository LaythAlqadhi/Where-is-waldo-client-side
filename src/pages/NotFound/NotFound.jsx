import React from 'react';
import style from './NotFound.module.css';

function NotFound() {
  return (
    <div className={style.container}>
      <h1>404 - Not Found</h1>
      <p>Sorry, the page you are looking for does not exist.</p>
    </div>
  );
}

export default NotFound;
