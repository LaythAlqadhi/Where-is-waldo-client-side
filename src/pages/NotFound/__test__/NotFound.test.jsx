import React from 'react';
import { render, screen } from '@testing-library/react';
import NotFound from '../NotFound';

it('renders NotFound component correctly', async () => {
  render(<NotFound />);

  expect(screen.container).toMatchSnapshot();
});
