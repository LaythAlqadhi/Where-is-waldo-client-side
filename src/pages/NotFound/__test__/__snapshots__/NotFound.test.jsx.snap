// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders NotFound component correctly 1`] = `undefined`;
